### PR TITLE
Amazon-AWS-EC2-Volumes

### PR DESCRIPTION
**Pull request by mihaitusa**
Ops sl's.
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@Bonczidai please review it!

**Comment by mihaitusa** [comment]
We will merge it after release to MC on: https://github.com/CloudSlang/score-actions/pull/133

File: content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        private: true
+    - region:
+        default: 'us-east-1'
+        required: false
+    - volume_id
+    - volumeId: ${volume_id}
+    - instance_id
+    - instanceId: ${instance_id}
+    - device_name
+    - deviceName: ${device_name}
```
Comment (Bonczidai) [inline content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl]: shouldn't you apply in these cases the usual pattern?

```
+    - proxyHost:
 +        default: ${get("proxy_host", "")}
 +        private: true
```

File: content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  java_action:
+    gav: 'io.cloudslang.content:score-jClouds:0.0.4'
+    class_name: io.cloudslang.content.jclouds.actions.volumes.AttachVolumeInRegionAction
+    method_name: execute
+
+  outputs:
+    - return_result: ${returnResult}
+    - return_code: ${returnCode}
+    - exception: ${exception if exception in locals() else ''}
```
Comment (Bonczidai) [inline content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl]: `get`?

**Comment by Bonczidai** [comment]
test cases

**Comment by Bonczidai** [comment]
:+1: -- please check comments

File: content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl (hunk 1 above)
Comment (mihaitusa) [inline content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl]: @Bonczidai  here the device_name is a required input...

File: content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl (hunk 2 above)
Comment (mihaitusa) [inline content/io/cloudslang/cloud/amazon_aws/volumes/attach_volume_in_region.sl]: @Bonczidai done

**Pull request merged**